frontend/src/app/pagenotfound/pagenotfound.component.html

```diff
@@ -0,0 +1,4 @@
+<div>
```

Comment:
can we add any css to this this is just a h1 with double content which btw goes against  w3c concept about schemantic element useage.

Reply:
My mistake, added css and redesigned html.